.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -15,14 +15,17 @@ env:
   ASDF_MAGE_VERSION: 1.15.0
 
 steps:
-  - group: "Mandatory Tests"
-    key: "xpack-winlogbeat-mandatory-tests"
+  - group: "x-pack/Winlogbeat Mandatory Tests"
+    key: "x-pack-winlogbeat-mandatory-tests"
     steps:
 
-      - label: ":windows: Xpack/Winlogbeat Win-2019 Unit (MODULE) Tests"
+      - label: ":windows: x-pack/Winlogbeat Win-2019 Unit (MODULE) Tests"
         key: "mandatory-win-2019-module-unit-tests"
         command: |
-          Set-Location -Path x-pack/winlogbeat
+          Import-Module .\.buildkite\scripts\changesets.psm1
+          defineModuleFromTheChangeSet 'x-pack\winlogbeat'
+          Write-Output "~~~ Will run tests with env var MODULE=$Env:MODULE"
```

Comment:
shouldn't this be $$Env:MODULE (IIRC a single `$` will be interpolated by Buildkite see https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation)